q0_utils.py

```diff
@@ -7,7 +7,7 @@
 from typing import Any, Dict, List, Optional
 
 import numpy as np
-from lcls_tools.common.data_analysis.archiver import Archiver
+from lcls_tools.common.data_analysis.archiver import *
```

Comment:
Ah I generally try to stay away from start imports but what I think you can do is just delete it from this file and import it directly in the other. Sorry I missed it in my last review comments